packages/components/src/components/animated-list/animated-list.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { motion, AnimatePresence } from 'framer-motion';
+import classNames from 'classnames';
+import './animated-list.scss';
+
+type TAnimatedListProps = {
+    className?: string;
+    children: JSX.Element[];
+};
+
+const animations = {
+    initial: {
+        height: 0,
+        opacity: 0,
+    },
+    animate: {
+        height: 'auto',
+        opacity: 1,
+        transition: {
+            height: {
+                duration: 0.2,
+            },
+            opacity: {
+                duration: 0.15,
+                delay: 0.05,
+            },
+        },
+    },
+    exit: {
+        height: 0,
+        opacity: 0,
+        transition: {
+            height: {
+                duration: 0.2,
+            },
+            opacity: { duration: 0.1 },
+        },
+    },
+};
+
+const AnimatedList = ({ className, children }: TAnimatedListProps) => {
```

Comment:
rename MessageList to AnimatedList to make it more generic

Reply:
I think the name AnimatedList is a bit vague, maybe we should specifically mention what the animation is. Like how we have the `FadeWrapper` component. What do you think?